samples/packages/asset-import/src/test/kotlin/com/atlan/pkg/aim/DeletedLinkTest.kt

```diff
@@ -0,0 +1,167 @@
+/* SPDX-License-Identifier: Apache-2.0
+   Copyright 2023 Atlan Pte. Ltd. */
+package com.atlan.pkg.aim
+
+import AssetImportCfg
+import com.atlan.model.assets.Asset
+import com.atlan.model.assets.Connection
+import com.atlan.model.assets.Database
+import com.atlan.model.assets.Link
+import com.atlan.model.assets.Schema
+import com.atlan.model.enums.AtlanConnectorType
+import com.atlan.model.fields.AtlanField
+import com.atlan.pkg.PackageTest
+import com.atlan.pkg.Utils
+import java.nio.file.Paths
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+
+private const val LINK_NAME = "hp"
+
+private const val LINK_URL = "https://hp.com"
+
+private const val DB_NAME = "DB"
+
+/**
+ * Test that after a link on an asset has been soft deleted and it's imported it becomes active
+ */
+class DeletedLinkTest : PackageTest("dlt") {
+    override val logger = Utils.getLogger(this.javaClass.name)
+
+    private val conn1 = makeUnique("c1")
+    private val conn1Type = AtlanConnectorType.IBM_DB2
```

Comment:
Need a distinct connector type within _at least_ the asset import tests (ideally all integration tests), otherwise this will potentially cause race condition / conflicts across the tests running in parallel.